src/common/lib/utils.ts

```diff
@@ -170,6 +172,21 @@ const utils = {
         });
     });
   },
+  getBoostagramFromInvoice: (
+    custom_records:
+      | {
+          "696969": string;
```

Comment:
What else can be in here? Do we need to care now?

Reply:
I think the following are interesting for us to show: 

* 32768:  https://github.com/lightning/blips/blob/master/blip-0002.md#messages
* 7629169: https://github.com/lightning/blips/blob/master/blip-0010.md

I think others can be ignored/not shown.

@kiwiidb what do you say?